_data/simple-icons.json

```
@@ -13262,6 +13262,11 @@
             "hex": "EC4A3F",
             "source": "https://rollupjs.org"
         },
+        {
+            "title": "Rook",
+            "hex": "2AC6EA",
+            "source": "https://rook.io"
```

There is an official Rook artwork repository, which we should use as our primary source. I've also added Linux Foundation trademark terms and the license for the icon. You may need to rebuild the icon from this new source too.

```suggestion
            "source": "https://github.com/rook/artwork/blob/d252db9f43753e6abf64db0ce12d38916a5de464/logo/Rook_Icon.svg",
            "guidelines": "https://www.linuxfoundation.org/legal/trademark-usage",
            "license": {
                "type": "Apache-2.0"
            }
```

---

SVG and icon data are updated